client/packages/android/README.MD

```diff
@@ -0,0 +1,103 @@
+# Mobile App with Discovery
+
+The client app is bundled with the server, on Android app can run as standalone site with a local server, as a client connecting to another server or as a server for other clients.
+On android, the server always starts and this allows for one mobile app for both client and client/server. 
+The mobile app will broadcast via dns sd and will be reachable by both Mobile and Electron clients on the local network. 
+We are using Capacitor to make use of existing optimisations and large library of native plugins. 
+All UI is served by bundled server, including discovery.
```

Comment:
When connected to another server the UI is served through the connected server isn't it?

Reply:
ohh yeah, thanks: 7693ec6c12347c61ab91645965ca53b7601de6cc, not as clear as I wanted, but should make sense